### PR TITLE
Products

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Please review my product-page which contains filter, context of my e-com website.

### CODE REVIEW

File: src/cave-components/Products/products.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+const { cartState, cartDispatch } = useCart();
+
+useEffect(() => {
+filterState.showData.length === 0 &&
+filterDispatch({type : "SET_DATA", payload: products});
+}, [products]);
+
+const filterCategory = (showData, filterOnCategories) => {
+if (filterOnCategories.length > 0) {
+console.log(filterOnCategories);
```
Comment (zoyatanweer) [inline src/cave-components/Products/products.jsx:25]: remove console.log

File: src/cave-components/Products/products.jsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+return showData.filter((product) => {
+console.log(product);
```
Comment (zoyatanweer) [inline src/cave-components/Products/products.jsx:27]: remove console.log

**Review by zoyatanweer — APPROVED** [review]
- clean code
- good to merge

**Review by srishtigautam21 — APPROVED** [review]
The code looks clean and understandable
Suggestion:
Include screenshots of UI in your description if no preview link is present
This helps the reviewer to better visualize your code

File: src/cave-components/Products/products.jsx (hunk 1 above)
Comment (Megha-Joshi) [reply src/cave-components/Products/products.jsx:25]: Okay

**Pull request merged**